### PR TITLE
De-Crapifies the sec taser - by buffing it

### PR DESCRIPTION
**Pull request by FlamingLily**
_"One last job."_

Reverts Skyrat-SS13/Skyrat-tg#6364

Made in conjuction with https://github.com/Skyrat-SS13/Skyrat-tg/pull/6926

Sec has no viable non-lethal options. Now they do.

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 1]
```diff
@@ -975,12 +975,5 @@
 	righthand_file = 'modular_skyrat/modules/aesthetics/guns/icons/guns_righthand.dmi'
 	icon_state = "armataser"
 	inhand_icon_state = null //so the human update icon uses the icon_state instead.
-	ammo_type = list(/obj/item/ammo_casing/energy/electrode/crap)
+	ammo_type = list(/obj/item/ammo_casing/energy/electrode)
```
Comment (capsaicinz) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm:978]: you forgot to add the //based comment here.

**Comment by lyricalpaws** [comment]
Before this gets closed, I'd like to remind people the taser got crapified because an antag got shot with it once and threw a shitfit.

**Comment by FlamingLily** [comment]
> 
> 
> Before this gets closed, I'd like to remind people the taser got crapified because an antag got shot with it once and threw a shitfit.

Hey now, remember that github is only for technical review, even though I agree.

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm (hunk 1 above)
Comment (lyricalpaws) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm:978]:
```suggestion
	ammo_type = list(/obj/item/ammo_casing/energy/electrode) //based
```

**Comment by Gandalf2k15** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/6921 - Incorporated into this, I've replaced the taser with a disabler.

**Pull request closed without merging**